sdk/src/main/java/io/opentelemetry/sdk/resources/EnvVarResource.java

```diff
@@ -75,4 +78,35 @@ static Attributes parseResourceAttributes(@Nullable String rawEnvAttributes) {
       return attrBuilders.build();
     }
   }
+
+  /** Builder utility for this EnvVarResource. */
+  public static class Builder extends ConfigBuilder<Builder> {
+    private static final String OTEL_RESOURCE_ATTRIBUTES_ENV = "OTEL_RESOURCE_ATTRIBUTES";
```

Comment:
the nice thing about using `ConfigBuilder` is that you only need to handle system property style names, and it will automatically translate those to env var style names when reading env vars (via `NamingConvention`)
```suggestion
```